subscription/forms.py

```diff
@@ -9,3 +9,10 @@ class CSVUploader(forms.Form):
     def save(self):
         csv_data = StringIO(self.cleaned_data["csv"].read())
         ingest_csv.delay(csv_data, self.cleaned_data["message_set"])
+
+
+class OptOutCSVUploader(forms.Form):
+    csv = forms.FileField()
+    def save(self):
+        csv_data = StringIO(self.cleaned_data["csv"].read())
+        ingest_opt_opts_csv.delay(csv_data)
```

Comment:
Does a `StringIO()` object survive via serialisation & celery? Wouldn't it be more straight forward to just send through the string you get back from `.read()` ?